src/spoof.ts

```diff
@@ -82,8 +82,10 @@ export interface GhostCursor {
 }
 
 // Helper function to wait a specified number of milliseconds
-const delay = async (ms: number): Promise<void> =>
+const delay = async (ms: number): Promise<void> => {
+  if (ms < 1) return
```

Comment:
according to `setTimeout` docs, if `ms < 1`, it uses `1`. So we don't want to use `setTimeout` if 0 is passed here.